api/src/main/java/org/openmrs/module/pihcore/setup/HtmlFormSetup.java

```
@@ -20,4 +29,26 @@ public static void setupHtmlFormEntryTagHandlers() throws Exception {
         htmlFormEntryService.addHandler(PihCoreConstants.HTMLFORMENTRY_FAMILY_HISTORY_RELATIVE_CHECKBOXES_TAG_NAME, new FamilyHistoryRelativeCheckboxesTagHandler());
     }
 
+    public static void loadHtmlForms() {
+
+        ResourceFactory resourceFactory = ResourceFactory.getInstance();
+        FormService formService = Context.getFormService();
+        HtmlFormEntryService htmlFormEntryService = Context.getService(HtmlFormEntryService.class);
+
+        try {
+            File htmlformDir = new File(PihCoreUtil.getFormDirectory());
+            Collection<File> files = FileUtils.listFiles(htmlformDir, null, true);
+
+            for (File file : files) {
+                try {
+                    HtmlFormUtil.getHtmlFormFromUiResource(resourceFactory, formService, htmlFormEntryService, "file:" + file.getAbsolutePath());
+                } catch (IOException e) {
+                    log.error("Unable to load HTML Form at path: " + file, e);
+                }
```

Same comment here as below - why are we swallowing this error?